.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: php
+
+php:
+  - 5.6
+
+before_script:
+  - mkdir -p build/logs
+  - composer self-update
+  - composer install --prefer-source
+
+script:
+  - ./vendor/bin/phpunit --coverage-clover ../build/logs/clover.xml
+  - ./vendor/bin/phpcs --standard=PSR2 ./src/ ./test/ZfcUserSettingsTest/ -n
+  - ./vendor/bin/test-reporter --stdout > codeclimate.json
+  - "curl -X POST -d @codeclimate.json -H 'Content-Type: application/json' -H 'User-Agent: Code Climate (PHP Test Reporter v1.0.1-dev)' https://codeclimate.com/test_reports"
```

Comment:
added <code>.travis.yml</code> config, need repo token in travis if using test-reporter.